janusgraph-dist/src/assembly/static/conf/gremlin-server/gremlin-server-berkeleyje.yaml

```diff
@@ -0,0 +1,40 @@
+host: localhost
```

Comment:
0.0.0.0 would be nice, so you can expose port 8182 to outside docker container, and connect using websockets for testing clients outside the container.